code/addons/actions/src/types.ts

```diff
@@ -0,0 +1,34 @@
+export interface ActionsParameters {
+  /**
+   * Actions configuration
+   *
+   * @see https://storybook.js.org/docs/essentials/actions#parameters
+   */
+  actions: {
+    /**
+     * Create actions for each arg that matches the regex.
+     *
+     * This is quite useful when your component has dozens (or hundreds) of methods and you do not
+     * want to manually apply the fn utility for each of those methods. However, this is not the
+     * recommended way of writing actions. That's because automatically inferred args are not
+     * available as spies in your play function. If you use argTypesRegex and your stories have play
+     * functions, you will need to also define args with the fn utility to test them in your play
+     * function.
+     *
+     * @example ArgTypesRegex: '^on.*'
+     */
+    argTypesRegex?: string;
+
+    /** Turn off this addon's behavior */
+    disable?: boolean;
+
+    /**
+     * Binds a standard HTML event handler to the outermost HTML element rendered by your component
```

Comment:
Is this only for the HTML framework? Shouldn't we be more specific about when you should & shouldn't use it?

Reply:
I have no idea, I just used [the docs as reference](https://storybook.js.org/docs/essentials/actions#handles) (they are not renderer specific). Honestly, I tried this in HTML and web components and nothing worked! 🤷

Reply:
TIL it will only work in tandem with manually specifying the `withActions` decorator

Reply:
It also works with other renderers e.g. React